client/views/view_units.cpp

```diff
@@ -607,39 +608,33 @@ void get_units_waiting_data(struct unit_waiting_entry *entries,
   if (nullptr == client.conn.playing) {
     return;
   }
+  const struct unit_type *ut; // Init unit_type struct
+  int pcity_near_dist = 0;    // Init distance
+  QString city = "";          // Init city name string
+  time_t dt = 0;              // Init date time
```

Comment:
I don't think these declarations are used outside of the `{}` block below, so I suggest limiting their scope to it
```suggestion
```